# HG changeset patch
# User Torsten <ttl@justmail.de>
# Date 1385219100 -3600
#      Sat Nov 23 16:05:00 2013 +0100
# Node ID 79e7c468d39247d7c93d5ef2dba5c204f52f637f
# Parent  4d06a98d820fc8ca046538be4e92611261e76a04
* files-dock-widget.cc (ctor): smaller height of the directory combo box

diff --git a/libgui/src/files-dock-widget.cc b/libgui/src/files-dock-widget.cc
--- a/libgui/src/files-dock-widget.cc
+++ b/libgui/src/files-dock-widget.cc
@@ -85,17 +85,17 @@ files_dock_widget::files_dock_widget (QW
 
   _current_directory = new QComboBox (_navigation_tool_bar);
   _current_directory->setToolTip (tr ("Enter the path or filename"));
   _current_directory->setEditable (true);
   _current_directory->setMaxCount (MaxMRUDirs);
   _current_directory->setInsertPolicy (QComboBox::NoInsert);
   _current_directory->setSizeAdjustPolicy (
     QComboBox::AdjustToMinimumContentsLengthWithIcon);
-  QSizePolicy sizePol (QSizePolicy::Expanding, QSizePolicy::Preferred);
+  QSizePolicy sizePol (QSizePolicy::Expanding, QSizePolicy::Maximum);
   _current_directory->setSizePolicy (sizePol);
 
   QAction *directory_up_action = new QAction (QIcon (":/actions/icons/up.png"),
                                               "", _navigation_tool_bar);
   directory_up_action->setToolTip (tr ("Move up one directory"));
 
   _sync_browser_directory_action
     = new QAction (QIcon (":/actions/icons/reload.png"),
